### DISCUSSION TITLE
Enforced allowedVersion only if it already satisifies the rule

### DISCUSSION THREAD

**Discussion by mikethecalamity** [renovatebot/renovate#29365] [Request Help]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

Currently I am using something like `"allowedVersions": "<24.5.0"`. Is there  a way to enforce this rule if, and only if, the version is already satisfying that rule? For example, if a version is manually updated to `25.0.1` in a specific project, the Renovate scans can suggest `25.1.0` because it already has surpassed the version in the restriction.

**Comment by rarkins** [comment]
Try using matchCurrentVersion

**Comment by mikethecalamity** [comment]
That worked, thanks!
